### PR TITLE
CM-69: added graphQL for fetching the definitions how to build filters

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openimis.atlassian.net/browse/CM-69

I Added graphQL query to obtain the possible way of building custom filter for BenefitPlan objects (for particular instance of BenefitPlan identified by uuid of benefit plan). 

Screenshot from 2023-06-01 16-31-13
based on such above query backend should return such kind of output. It is based on such kind of schema provided by such BenefitPlan:
Screenshot from 2023-06-01 16-06-59

Moreover I changed in named tuple 'value' into 'type'. Type will be simply the type of particular field for example string, boolean, integer, double etc.

Mocked example how gql query can work on another kind of object (Policy)
Screenshot from 2023-06-01 16-25-28

### CODE REVIEW

File: core/schema.py [hunk 1]
```diff
@@ -805,6 +813,53 @@ def resolve_modules_permissions(self, info, **kwargs):
                 ))
         return ModulePermissionsListGQLType(list(config))
 
+    def resolve_custom_filters(self, info, **kwargs):
+        # TODO - add permission
```
Comment (sniedzielski) [inline core/schema.py:817]: Should we create special permission for such graphQL?

**Comment by delcroip** [comment]
still a draft ?

**Comment by sniedzielski** [comment]
yes @delcroip. Because First we need to merge this PR https://github.com/openimis/openimis-be-core_py/pull/181. I'm about to push changes after Damian's review.

**Comment by sniedzielski** [comment]
@dborowiecki You can start checking PR. I left one questuion regarding GQL permission for such query with fetching the way how to build filters

File: core/schema.py (hunk 1 above)
Comment (dborowiecki) [reply core/schema.py:829]: I don't think it's required, if any restrictions were to be added they should be on specific custom filter level not mutation. Just validating if user exists is enough.

**Review by dborowiecki — CHANGES_REQUESTED** [review]
Please add checking if user is logged in inside the gql queyr

File: core/schema.py (hunk 1 above)
Comment (sniedzielski) [reply core/schema.py]: ok, thanks for clarification

**Review by dborowiecki — APPROVED** [review]

**Pull request merged**